pkg/volume/fc/attacher.go

```diff
@@ -77,7 +77,7 @@ func (attacher *fcAttacher) VolumesAreAttached(specs []*volume.Spec, nodeName ty
 func (attacher *fcAttacher) WaitForAttach(spec *volume.Spec, devicePath string, _ *v1.Pod, timeout time.Duration) (string, error) {
 	mounter, err := volumeSpecToMounter(spec, attacher.host)
 	if err != nil {
-		klog.Warningf("failed to get fc mounter: %v", err)
+		klog.InfoS("Failed to get fc mounter", "err", err)
```

Comment:
I know this isn't your change, but I prefer the style of letting the caller log the error instead of logging and then returning the error.

Can we change places like this to something like `return fmt.Errorf("failed to get fc mounter: %w", err)`

Reply:
Hi, @msau42, thanks for your suggestion. I have modified this place to `return fmt.Errorf("failed to get fc mounter: %w", err)`.
But, if the caller doesn't handle the returned error, will it go against for trouble-location? Such as the following modification `klog.Warningf("failed to get fc mounter: %v", err)` in func `GetDeviceMountPath`.

Reply:
Most places I am aware of should be handling the error. If not, then that should be fixed. Where are you seeing the caller not handling the error? For `GetDeviceMountPath` I see that all callers are handling the error properly: https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/operationexecutor/operation_generator.go